integration_tests/test_list_count.py

```diff
@@ -0,0 +1,33 @@
+from lpython import i32
+
+def test_list_count():
+    i: i32
+    x: list[i32] = []
```

Comment:
What about testing with `list[str]`, `list[tuple[i32, str, i64]`, etc.?

Reply:
I think this is done.